libafl/src/events/llmp.rs

```diff
@@ -1439,17 +1479,19 @@ where
                     if child_status == 137 {
                         // Out of Memory, see https://tldp.org/LDP/abs/html/exitcodes.html
                         // and https://github.com/AFLplusplus/LibAFL/issues/32 for discussion.
+                        let _ = notify_death(client_id, self.broker_port);
```

Comment:
we can move this notify before the if, that way we don't need to do it on both code paths